### ISSUE TITLE
[joss] Need locations of benchmarking data!

### ISSUE THREAD

**Issue by sneakers-the-rat** [juanmc2005/diart#175]
Trying to find the data to run the benchmarks, and I can't find all the source data:

- [x] **VoxConverse** - found it!
- [ ] **AMI** - found the download page https://groups.inf.ed.ac.uk/ami/download/ but I can't tell what files I need for the benchmark
- [ ] **DIHARD III* - can't find download
- [ ] **DIHARD II* - can't find download

this should all be in the docs!

part of https://github.com/openjournals/joss-reviews/issues/5266

**Comment by sneakers-the-rat** [comment]
Forgot to say - I also couldnt tell easily from the text of the repo paper which samples I should use from voxconverse, might be nice to have a link to a source and a little script for loading them - making reproducing like 1 step if you have a spare half hour or so

**Comment by juanmc2005** [comment]
@sneakers-the-rat good idea! However, the two DIHARD datasets are private. One has to ask for permission to use them.
For AMI we used Mix-Headset (downloadable [here](https://github.com/pyannote/AMI-diarization-setup/tree/main/pyannote))
